design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
+    ): UpdatedPricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    assignBillingAddressToNegotiableQuote(
+        billingAddress: NegotiableQuoteBillingAddressInput!
```

Comment:
It seems confusing that this argument name is `billingAddress` but it has a `cartId` property on it.

Reply:
As per this comment https://github.com/magento/architecture/pull/381#discussion_r431437186, it is not possible to set Billing Address on NQ.